### DISCUSSION TITLE
请问ipv6 only的服务器如何连接？

### DISCUSSION THREAD

**Discussion by darenbooks** [lollipopkit/flutter_server_box#98] [Q&A]
谢谢，你们的软件既好看又好用。就是遇到了一个问题，only ipv6 的服务器，怎么连接？
不管是直接填入数字 xxxx:xxxx:xxxx::xx ,还是带括号的 [xxxx:xxxx:xxxxx::xx] 格式的地址，都无法连接上，错误显示 alterUrl is null 。
是我填写的格式的问题吗？还是设置的问题？
请问该怎么解决？

我的系统是 ios 14 和 ios 15 。

**Comment by lollipopkit** [comment]
无法使用ipv6的原因可能有挺多，可能是当前路由器不支持等。

我经过测试发现
- 直接输入ipv6地址
- 输入指向ipv6的域名

都是可以正常连接的。

**Reply by darenbooks** [reply]
感谢回答，经过排查，果然是我自己的代理软件的问题造成的。谢谢。
